src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/StandardDirectionOrderDJ.java

```diff
@@ -200,18 +200,18 @@ private CallbackResponse populateDisposalTrialScreen(CallbackParams callbackPara
                                                             + " for a disposal hearing")
                                                 .build());
 
-        caseDataBuilder.disposalHearingMedicalEvidenceDJ(DisposalHearingMedicalEvidence
-                                                             .builder()
-                                                             .input1("The claimant has permission to rely upon the"
-                                                                         + " written expert evidence served with the"
-                                                                         + " Particulars of Claim to be disclosed "
-                                                                         + "by 4pm")
-                                                             .date1(LocalDate.now().plusWeeks(4))
-                                                             .input2("and any associated correspondence and/or "
-                                                                         + "updating report disclosed not later "
-                                                                         + "than 4pm on the")
-                                                             .date2(LocalDate.now().plusWeeks(4))
-                                                             .build());
+        // caseDataBuilder.disposalHearingMedicalEvidenceDJ(DisposalHearingMedicalEvidence
+        // .builder()
+        // .input1("The claimant has permission to rely upon the"
+        //      + " written expert evidence served with the"
+        //      + " Particulars of Claim to be disclosed "
+        //      + "by 4pm")
+        // .date1(LocalDate.now().plusWeeks(4))
+        // .input2("and any associated correspondence and/or "
+        //      + "updating report disclosed not later "
+        //      + "than 4pm on the")
+        // .date2(LocalDate.now().plusWeeks(4))
+        // .build());
 
```

Comment:
As this code is in git, if it's not necessary then we can remove it instead of commenting it out. If we want it to stay there for a reason, add a comment explaining why so that another dev doesn't remove it as unnecessary.